module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -3487,4 +3552,91 @@ protected function checkTitleHoldRules(array $bib, array $patron): bool
         }
         return false;
     }
+
+    /**
+     * Gets title information for holds placed in an INN-Reach system
+     *
+     * @param $holdId the id of the hold from Sierra
+     * @param $bibId  the id of the bib from Sierra
+     *
+     * @return array|void
+     *
+     * @throws ILSException
+     */
+    protected function getInnReachHoldTitleInfoFromId($holdId, $bibId)
+    {
+        if (!isset($this->innReachDb)) {
+            $this->getInnReachDb();
```

Comment:
Maybe rather than making getInnReachDb a void function, it would be better to have it actually return the database connection... then it could contain logic like:

```
if (!isset($this->innReachDb)) {
    // ... establish connection here ...
}
return $this->innReachDb;
```

And then the database code below can use the returned connection. I think that slightly clarifies the flow of everything.